lib/Cro/HTTP/Client.pm6

```diff
@@ -587,6 +589,18 @@ class Cro::HTTP::Client {
                 }
             }
         }
+
+        # Set User-agent, check if wasn't set already for us
+        unless $request.has-header('User-agent') {
+            if %options<user-agent>:exists {
+                with %options<user-agent> {
+                    $request.append-header('User-agent', $_) if $_;
+                }
+            } else {
```

Comment:
Probably we want an `elsif` here that is doing the instance-level check.

Reply:
This still applies; if you say at the client-level you want no user agent, then we should respect that.